test/wiki-feed-parser.test.js

```diff
@@ -54,15 +54,15 @@ test('Testing wiki-feed-parser.getData with no pre tag', async () => {
 
   fetch.mockResponseOnce(mockBody);
 
-  await parser.getData().catch(err => {
-    expect(err).toStrictEqual(noPreErr);
+  await parser.getData().catch(error => {
```

Comment:
This file can get reverted, these are all fine.  It's only in cases where we log it.